packages/builders/__tests__/util.test.ts

```diff
@@ -12,3 +12,18 @@ describe('validation', () => {
 		expect(isValidationEnabled()).toBeFalsy();
 	});
 });
+
+describe('normalizeArray', () => {
+	test('normalizes an array or array (when input is an array)', () => {
+		expect(normalizeArray([[1, 2, 3]])).toEqual([1, 2, 3]);
+	});
+
+	test('normalizes an array (when input is rest parameter)', () => {
+		expect(normalizeArray([1, 2, 3])).toEqual([1, 2, 3]);
+	});
+
+	test('always returns a clone', () => {
```

Comment:
Add a test case for rest too

Reply:
Rest doesnt gets cloned in normalizeArray as the original input wasn't an array but rest params